component-library/src/app/pages/date-picker-documentation/date-picker-doc-code.component.scss

```diff
@@ -0,0 +1,5 @@
+.datePicker-container {
+  max-width: 320px;
```

Comment:
max-width in design spec is 380px, currently set at 320px causes the interactive demo component to be off center. I know you're still working on the interactive demo but want to flag it.

Reply:
Good catch Bobby, thanks
So we have to update it for Input and Banner as well

Reply:
Done! Thanks :)